.github/workflows/tests.yml

```diff
@@ -143,38 +143,3 @@ jobs:
         else
             ./run_test.sh
         fi
-
-  test-vm:
```

Comment:
@scaronni you probably already know you can't really test kernel modules inside a container. This VM testing portion goes one step further than the container portion and appears to actually install the kernel modules and load them. IMO without this VM step it drops the test coverage in a pretty significant way.

Reply:
Ubuntu 20.04 will go EOL April next year so maybe it makes sense to try updating to 24.04? Or is the general strategy here to test older distro and then you'd expect newer distros to also work (somewhat by induction)?

Reply:
Unless you have the hardware for the modules available to the VM, even loading the modules in the VM does not bring anything. I think we should test that DKMS is just functionally correct.

Reply:
No we don't have any strategy for the VMs, it's just that matching the comment above, I don't see the value of having both the container and the VM.